frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -40,6 +40,8 @@ export class FeedItemWithPreview extends React.Component {
 
     this.state = {
       invalidMentionList: [],
+      commentsShown: false,
+      commentText: '',
```

Comment:
Maybe a better naming will be something like
```suggestion
      isDiscussionDisplayed: false,
      discussionToggleButtonLabel: '',
```
If you change here, you need to change everywhere. And if you have better ideas, go for it ! I just don't like "commentText" because it seems that you are talking about the real comment text, not the button.

Reply:
Is the state discussionToggleButtonLabel used somewhere? and for the commentsShown even if you leave it like that I would put "is" or "are" in front of it, it's a good naming practice to say it's a boolean. "areCommentsShown" or "isCommentListShown".

Reply:
I made the change thanks